recipes-nodes/rbuilder/rbuilder.inc

```diff
@@ -16,14 +16,15 @@ BINDGEN_EXTRA_CLANG_ARGS = "--sysroot=${WORKDIR}/recipe-sysroot -I${WORKDIR}/rec
 
 FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"
 SRC_URI += "file://init"
-SRC_URI += "file://config"
+SRC_URI += "file://config.mustache"
 SRC_URI += "file://ofac.json"
 S = "${WORKDIR}/git"
 
 INITSCRIPT_NAME = "rbuilder"
-INITSCRIPT_PARAMS = "defaults 99" 
+INITSCRIPT_PARAMS = "defaults 99"
 
 FILES:${PN} += "${sysconfdir}/init.d/rbuilder"
+FILES:${PN} += "${sysconfdir}/rbuilder.config.mustache"
 FILES:${PN} += "${sysconfdir}/rbuilder.config"
```

Comment:
you need to remove this 
```suggestion

```

Reply:
See my comment about the pre-created config. We should still need it here in that case, right?

Reply:
This here would make sure that the built image would have rbuilder.config file too.
But we are only creating that during runtime after we fetch the config and render it, right?

That's why we can only ensure the existence of rbuilder.config.mustache during build time